public/js/script.js

```diff
@@ -6,4 +6,181 @@ for (let i = 0; i < createImageContainers.length; i++) {
        createImageContainers[i].style.backgroundImage = 'url(' + url + ')';
        createImageContainers[i].style.backgroundSize = 'cover';
    }
-}
\ No newline at end of file
+}
+
+window.addEventListener('load', () => {createEnableDisable(); createDisableAll();});
+window.addEventListener('keydown', () => {createEnableDisable();});
+window.addEventListener('click', () => {createEnableDisable();});
```

Comment:
These listeners should be on individual elements and not the whole page, especially since this script is loaded for all pages of the website.